### PR TITLE
Garden Biogenerator

### PR DESCRIPTION
**Pull request by Geevies**
* Added a minified biogenerator to the surface garden. It is quite inefficient, but you can move into its tile.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/game/machinery/biogenerator.dm [hunk 1]
```diff
@@ -639,4 +639,32 @@
 			man_rating += P.rating
 
 	build_eff = man_rating
-	eat_eff = bin_rating
\ No newline at end of file
+	eat_eff = bin_rating
+
+/obj/machinery/biogenerator/small
```
Comment (MarinaGryphon) [inline code/game/machinery/biogenerator.dm:644]: This inherits component_types, meaning it'll *probably* be able to be dismantled and reassembled to get a normal one.

**Review by MarinaGryphon — CHANGES_REQUESTED** [review]

**Review by JohnWildkins — CHANGES_REQUESTED** [review]
what marina said

good otherwise but unless it's meant to be avant-garde powergamer bait it should probably have fewer and worse components

**Comment by Geevies** [comment]
> 
> good otherwise but unless it's meant to be avant-garde powergamer bait it should probably have fewer and worse components

the components aren't as effective in the smaller one, so yea

**Review by JohnWildkins — APPROVED** [review]

**Review by Alberyk — APPROVED** [review]

**Pull request merged**